Common/Orders/Fills/EquityFillModel.cs

```diff
@@ -30,17 +30,17 @@ namespace QuantConnect.Orders.Fills
     /// </summary>
     public class EquityFillModel : IFillModel
     {
-        /// <summary>
-        /// The parameters instance to be used by the different XxxxFill() implementations
-        /// </summary>
-        protected FillModelParameters Parameters { get; set; }
-
```

Comment:
Let's revert these changes 👍

Reply:
Seems to be reverted although github isnt showing such? https://github.com/aarjaneiro/Lean/blob/930d818d5878e57862fd073f464d6c2bbca25dd9/Common/Orders/Fills/FillModel.cs#L33